GRANTS.yaml

```diff
@@ -67,6 +67,17 @@ CINECA-Project:
     url: https://www.cineca-project.eu
 
 
+dekcd:
+    name: de.kcd
+    github: https://github.com/deKCD
```

Comment:
you can remove this line, by default it will assume that the key at the top of the entry (in your case `dekcd`) is also the github name (capitalization doesnt matter, so should work by just deleting this line)